src/audio/src/src.c

```diff
@@ -37,12 +37,24 @@
 #include <sof/audio/coefficients/src/src_std_int32_table.h>
 #endif
 
-#define trace_src(__e, ...) \
+/* tracing */
+#define trace_src(__e, ...)					\
 	trace_event(TRACE_CLASS_SRC, __e, ##__VA_ARGS__)
-#define tracev_src(__e, ...) \
+#define trace_src_with_ids(comp_ptr, __e, ...)			\
+	trace_event_comp(TRACE_CLASS_SRC, comp_ptr,		\
+			 __e, ##__VA_ARGS__)
+
+#define tracev_src(__e, ...)					\
 	tracev_event(TRACE_CLASS_SRC, __e, ##__VA_ARGS__)
-#define trace_src_error(__e, ...) \
+#define tracev_src_with_ids(comp_ptr, __e, ...)			\
```

Comment:
Please either align backslashes or use just place it right after content + 1 space

Reply:
Is is aligned, there is a problem with visualization in github